main.js

```diff
@@ -468,6 +609,11 @@ exports.onUnload = function (reason) {
       }
 
       win.VerticalTabs.unload();
+      if (win.fullScreen){
+        win.FullScreen.hideNavToolbox = win.oldHideNavToolbox;
+        win.FullScreen._updateToolbars = win.oldUpdateToolbars;
+      }
```

Comment:
We don't want to restore the functions if we start in fullscreen mode but don't end in fullscreen mode?  That seems a little odd…  (Or is it a typo where the if should check for `win.FullScreen`?)